openml_OS/views/pages/api_new/v1/xml/data-get.tpl.php

```diff
@@ -18,6 +18,7 @@
   <?php if ($version_label != null): ?><oml:version_label><?php echo htmlspecialchars($version_label); ?></oml:version_label><?php endif; ?>
   <?php if ($citation != null): ?><oml:citation><?php echo htmlspecialchars($citation); ?></oml:citation><?php endif; ?>
   <?php if( is_array( $tag ) ) foreach( $tag as $t ): ?><oml:tag><?php echo htmlspecialchars($t); ?></oml:tag><?php endforeach; ?>
+  <!-- <?php if( is_array( $studies ) ) foreach( $studies as $s ): ?><oml:study><?php echo $s; ?></oml:study><?php endforeach; ?> -->
```

Comment:
Please remove this. This can't be in the template because it will break some XML readers.